README.md

```diff
@@ -239,36 +155,324 @@ bash install.bash
 .\install.bat
 ```
 
-### Docker installs
-
-Refer to
-[these instructions](https://voxel51.com/docs/fiftyone/environments/index.html#docker)
-to see how to build and run Docker images containing source or release builds
-of FiftyOne.
-
 ### Generating documentation
 
 See the
 [docs guide](https://github.com/voxel51/fiftyone/blob/develop/docs/README.md)
 for information on building and contributing to the documentation.
 
-## Uninstallation
+### Uninstallation
 
 You can uninstall FiftyOne as follows:
 
 ```shell
 pip uninstall fiftyone fiftyone-brain fiftyone-db
 ```
 
-## Contributors
+
+</details>
+
+<div id='-prerequisites'/>
+
+<details>
+<summary>Prerequisites for beginners</summary>
```

Comment:
This section and it's child sections seem to verbose. Can we move some of this to the documentation instead then just link to it here?

Reply:
You are right; the idea of this section is to provide a solution for beginners. We can add a link to documents when the new documentation is ready.